src/components/OrgFile/components/AttributedString/index.js

```diff
@@ -71,7 +70,7 @@ export default ({ parts, subPartDataAndHandlers }) => {
       normalised = normalised.substr(3);
       dir = dir.match(/(.*)\//)[1];
     }
-    return dir + '/' + normalised;
```

Comment:
@tarnung I think this introduced a regression. If I'm in a folder `/org/a.org` which includes a link to `[[file:reference.org][Reference]]`, then the file will not be found:

It stays like this:

![image](https://user-images.githubusercontent.com/505721/100523029-77909780-31ad-11eb-9e90-2c07756b9486.png)
![image](https://user-images.githubusercontent.com/505721/100523038-85461d00-31ad-11eb-8004-d944ef37c4b6.png)

Was there a reason to remove the `dir`?

Reply:
Yes, there was. If i set the path in redux, there is no need for the initial `/file/`.
I'm sure I can solve it in a way that works for directories too.

Reply:
:+1: 

I like that you're rewriting it to use React and Redux instead of file links, too. That seems more ideomatic to me.

Great work as always!